specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManagerConnectivityConfiguration.json

```diff
@@ -0,0 +1,414 @@
+        "responses": {
+          "200": {
+            "description": "Get successful. The operation returns the specified Managed Network Connectivity Configuration resource.",
+            "schema": {
+              "$ref": "#/definitions/ConnectivityConfiguration"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ConnectivityConfigurationsGet": {
+            "$ref": "./examples/NetworkManagerConnectivityConfigurationGet.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "ConnectivityConfigurations"
+        ],
+        "operationId": "ConnectivityConfigurations_CreateOrUpdate",
+        "description": "The Put ConnectivityConfigurations operation creates/updates a new Managed Network Peering Configuration",
+        "parameters": [
+          {
+            "name": "connectivityConfiguration",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ConnectivityConfiguration"
+            },
+            "description": "Parameters supplied to create/update a Managed Network Peering Configuration"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Update successful. The operation returns the resulting Managed Network Peering Configuration resource.",
+            "schema": {
+              "$ref": "#/definitions/ConnectivityConfiguration"
+            }
+          },
+          "201": {
+            "description": "Create successful. The operation returns the resulting Managed Network Peering Configuration resource.",
+            "schema": {
+              "$ref": "#/definitions/ConnectivityConfiguration"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ConnectivityConfigurationsPut": {
+            "$ref": "./examples/NetworkManagerConnectivityConfigurationPut.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      },
+      "delete": {
+        "tags": [
+          "ConnectivityConfigurations"
+        ],
+        "operationId": "ConnectivityConfigurations_Delete",
+        "description": "The Delete ConnectivityConfigurations operation deletes a Managed Network Connectivity Configuration, specified by the  resource group, Managed Network name, and connectivity Configuration name",
+        "responses": {
```

Comment:
As this is a long running operation, add response 202 to the list. Refer https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#202-accepted-and-location-headers